### ISSUE TITLE
Error parsing log stream for Android 5.0.2

### ISSUE THREAD

**Issue by nwanglu** [openstf/adbkit-logcat#4]
Well, this is the same issue as #2, this library is great and it saved lots of time while I have to deal with ADB.
So is there any plan to support L in the future? Thanks.

**Environment**
- win7 x64
- node: v 4.2.3
- adbkit:  2.4.0
- Android version on Device: 5.0.2
- Android Debug Bridge version 1.0.32 Revision 09a0d98bebce-android

**CODE**

``` javascript
var adb = require('adbkit');
var moment = require('moment');
var client = adb.createClient();

//001198336888
//FIPZMZZP99999999
client.openLogcat('FIPZMZZP99999999', {
    clear: true,
}, function (err, reader) {
    reader.on('entry', function (entry) {
        // var line = moment(entry.date).format('MM-DD HH:mm:ss.SSS') + ' ' +
        //     entry.pid + '-' + entry.tid + '/? ' + entry.priority + '/' + entry.tag + ': ' + entry.message;
        // console.log(line.trim());
        console.log(entry);
    });

    reader.on('end', function () {
        console.log(moment(new Date()).format('MM-DD HH:mm:ss.SSS') + ' Logcat Reader Event - END');
    });

    reader.on('error', function (err) {
        console.log(moment(new Date()).format('MM-DD HH:mm:ss.SSS') + ' Logcat Reader Event - ERROR' + err);
    });

    reader.on('finish', function () {
        console.log(moment(new Date()).format('MM-DD HH:mm:ss.SSS') + ' Logcat Reader Event - FINISH');
    });
});
```

**output**
![image](https://cloud.githubusercontent.com/assets/11607990/13241675/3718cec0-da28-11e5-907d-970a7cf47d28.png)

**Comment by sorccu** [comment]
#3 could perhaps have fixed this, it depends on the device though. What's your device? The output of `adb devices -l` without the serial number would be useful.

**Comment by nwanglu** [comment]
C:\Users\xiaolu.wxl>adb shell getprop ro.build.version.release
5.0.2

C:\Users\xiaolu.wxl>adb shell getprop ro.build.version.sdk
21

C:\Users\xiaolu.wxl>adb devices -l
List of devices attached
FIPZMZZP99999999       device product:hennessy model:Redmi_Note_3 device:hennessy

C:\Users\xiaolu.wxl>adb shell getprop

[gsm.serial]: [81548Y024285                   PP        P        CIP:P     10P]

[gsm.version.baseband]: [MOLY.LR9.W1423.MD.LWTG.MP.V24.P51, 2016/01/19 19:40]
[gsm.version.ril-impl]: [mtk gemini ril 1.0]

]

[rild.libargs]: [-d /dev/ttyC0]

[ro.build.date]: [Sat Jan 30 03:23:20 CST 2016]
[ro.build.description]: [hennessy-user 5.0.2 LRX22G V7.1.8.0.LHNCNCK release-keys]

[ro.product.model]: [Redmi Note 3]

**Comment by sorccu** [comment]
It would be very helpful if you could run `adb shell 'logcat -dB > /data/local/tmp/bin.log' && adb pull /data/local/tmp/bin.log` and send me the log file. However, it can contain lots of private information, so keep that in mind.

Anyway it's not an Android L issue per se, some devices just do things a little differently. Usually Chinese devices for some reason.

**Comment by nwanglu** [comment]
Well, Log is sent to your mailbox s***@shoqolate.com.
Thanks for take your time to look at this issue!

**Comment by scottchejin** [comment]
Well, this  issue  fixed in Android L branch and develop branch, but when will merge it to master branch and publish to NPM?

**Comment by sorccu** [comment]
I have been very busy, and like the commit message in the branch says, there are issues with that branch. Will try to look into this soon, however it would help if people could post what works for them and what doesn't. Specifically emulators need testing.

**Comment by sorccu** [comment]
Ok I finally have a device where this issue occurs. Will publish shortly.

**Comment by sorccu** [comment]
Released as 1.1.0. Let me know how it works for you guys.
